### PR TITLE
JPERF-836 Fix browse boards selector

### PR DESCRIPTION
**Pull request by mgrzaslewicz**

### CODE REVIEW

File: src/main/kotlin/com/atlassian/performance/tools/jirasoftwareactions/api/page/BrowseBoardsPage.kt [hunk 1]
```diff
@@ -36,15 +36,15 @@ class BrowseBoardsPage(
             return emptyList()
         }
         driver.wait(
-            Duration.ofSeconds(5), 
-            elementToBeClickable(By.cssSelector("#ghx-manage-boards-filter a"))
+            Duration.ofSeconds(5),
+            elementToBeClickable(By.xpath("//*[@aria-controls='board-types']"))
```
Comment (dagguh) [inline src/main/kotlin/com/atlassian/performance/tools/jirasoftwareactions/api/page/BrowseBoardsPage.kt:40]: XPath is useful for complex hierarchical locators. Here a simple CSS selector is sufficient:
```suggestion
            elementToBeClickable(By.cssSelector("[aria-controls='board-types']"))
```

**Review by dagguh — CHANGES_REQUESTED** [review]
- [ ] changelog

**Comment by dagguh** [comment]
WRT red CI, it seems that the `BrowseBoardsPageMock` needs to include the ARIA attributes now. The mock was introduced [here](https://github.com/atlassian/jira-software-actions/pull/2) and I'm not so sure that mocking that is a good idea (not sure = could be good, could be bad). Let's try updating the mock and see how it pans out.

**Comment by dagguh** [comment]
We can reproduce JPERF-836 by adding the affected Jira version to the CI matrix.
PS. we can also add some assertions in `JiraSoftwareScenarioIT` to show that `BROWSE_BOARDS` is being run successfully.

**Comment by mgrzaslewicz** [comment]
> PS. we can also add some assertions in JiraSoftwareScenarioIT to show that BROWSE_BOARDS is being run successfully.
It already contained this action and the altered selector is being tested

**Comment by mgrzaslewicz** [comment]
> We can reproduce JPERF-836 by adding the affected Jira version to the CI matrix.

@dagguh What do you mean? We can already see plan failing
https://server-gdn-bamboo.internal.atlassian.com/browse/RELEASE-JPTDSR

**Review by dagguh — DISMISSED** [review]

**Review by ewefie — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -25,6 +25,9 @@ Dropping a requirement of a major version of a dependency is a new contract.
 
 ## [Unreleased]
 [Unreleased]: https://github.com/atlassian/jira-software-actions/compare/release-1.3.5...master
+[JPERF-836]: https://ecosystem.atlassian.net/browse/JPERF-836
+
+- Fix browse boards selector [JPERF-836]
```
Comment (pczuj) [inline CHANGELOG.md:30]: Nit: Issue key mention is inconsistent with the previous entries in the CHANGELOG. Not sure how much this matters.

**Review by pczuj — APPROVED** [review]

**Review by dagguh — APPROVED** [review]

**Review by pczuj — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (pczuj) [reply CHANGELOG.md:30]: The inner peace is probably the thing for me :D

I'm fine with leaving it as it is.

File: src/test/kotlin/com/atlassian/performance/tools/jirasoftwareactions/api/scenario/JiraSoftwareScenarioIT.kt [hunk 3]
```diff
@@ -33,16 +34,15 @@ class JiraSoftwareScenarioIT {
      */
     @Test
     fun shouldRunScenarioWithoutErrors() {
-        val version = System.getenv("JIRA_SOFTWARE_VERSION") ?: "8.5.3"
+        val version = System.getenv("JIRA_SOFTWARE_VERSION") ?: "8.20.13"
         logger.info("Testing Jira $version")
         val scenario = JiraSoftwareScenario()
         val metrics = mutableListOf<ActionMetric>()
-        val actionMeter = ActionMeter(
```
Comment (dagguh) [inline src/test/kotlin/com/atlassian/performance/tools/jirasoftwareactions/api/scenario/JiraSoftwareScenarioIT.kt:40]: Respect for fixing deprecations :heart:

**Review by dagguh — APPROVED** [review]

**Pull request merged**